salesagility/SuiteCRM
# Feature - REST API Meta calls

Task: SCRM-775

## Description



This change adds calls to get metadata information such as favorites, viewed, viewdefs etc.

## Motivation and Context

* prepares for version 8 of suitecrm

## How To Test This

* Setup composer acceptance tests (see tests/readme.md)
* run codecept run api

## Types of changes


### Final checklist
